packages/client/src/components/ToolItem/ToolItem.component.js

```diff
@@ -24,7 +21,10 @@ export const ToolItem = ({ tool }) => {
       if (isFavourite) {
         fetch(`${getApiBaseUrl()}/api/favourites`, {
           method: 'DELETE',
-          body: JSON.stringify({ user_id: userId, tool_id: id }),
+          body: JSON.stringify({
+            user_id: userId,
```

Comment:
This looks like a serious problem: What if you send `user_id: 'someone elses userid'`? Can you then change another user's favorites? The API should set the user_id automatically